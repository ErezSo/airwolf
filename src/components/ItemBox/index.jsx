import React from "react";

const styles = {
  box: {
    width: "100%",
    height: "100%",
    textAlign: "center"
  }
};

const ItemBox = ({ item, idx, itemOpacity }) => {
  const { box } = styles;
  return (
    <div style={{ ...box, ...itemOpacity }}>
      <img src={`assets/${item.img}`} alt={`item${idx}`} />
    </div>
  );
};

export default ItemBox;
