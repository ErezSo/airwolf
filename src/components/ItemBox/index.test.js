import React from "react";
import { shallow } from "enzyme";
import ItemBox from "./";

describe("ItemBox", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ItemBox />);
  });

  it("renders", () => {
    expect(wrapper).toHaveLength(1);
  });

  it("renders correctly", () => {
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
