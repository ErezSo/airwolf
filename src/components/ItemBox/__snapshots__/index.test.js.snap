// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemBox renders correctly 1`] = `
"<div style={{...}}>
  <img src=\\"assets/undefined\\" alt=\\"itemundefined\\" />
</div>"
`;
